ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1324,6 +1324,13 @@ router_bgp:
         activate: < true | false >
         route_map_in: < route_map_name >
         route_map_out: < route_map_name >
+  address_family_rtc:
+    peer_groups:
+      < peer_group_name >:
+        activate: < true | false >
+        default_route_target:
+          only: < true | false >
+          encoding_origin_as_omit: < true | false >
```

Comment:
Your Readme says `encording_origin_as_omit : < true | false >` but your test has `none`. The code also only looks if this knob is defined, so we could just change the readme to reflect that.

Reply:
Thanks, fixed!